app/src/androidTest/java/ch/epfl/sdp/drone3d/MainActivityTest.kt

```diff
@@ -63,14 +89,30 @@ class MainActivityTest {
     }
 
     @Test
-    fun goToMappingMissionSelectionWorks() {
+    fun goToMappingMissionSelectionWorksWhenActiveSession() {
+        val user = Mockito.mock(FirebaseUser::class.java)
+
```

Comment:
I prefere having static imports of Mockito.* ans Espresso .*

such that we only have 'when'() and onView() aso.


That's what I did in my tests, we should just choose one side and stick to it

Reply:
I fix it in the tests